iceoryx_posh/source/roudi/service_registry.cpp

```diff
@@ -119,7 +128,7 @@ void ServiceRegistry::find(ServiceDescriptionVector_t& searchResult,
 
             if (match)
             {
-                searchResult.emplace_back(*entry);
+                function(*entry);
```

Comment:
Same here

Reply:
No change needed, in functional programming (and math) everything that maps arguments to a result is a function ...